sernet.gs.service/src/sernet/gs/scraper/ZIPGSSource.java

```diff
@@ -183,7 +183,7 @@ public Node parseBausteinDocument(String bausteinFileName) throws GSServiceExcep
 			
 		
 		} catch (Exception e) {
```

Comment:
Why do you switch from _error_ to _warn_ here and why do you not log the stack trace anymore? Please use the _Logger LOG_ defined in the class.